apps/progs.c

```diff
@@ -44,7 +44,7 @@ FUNCTION functions[] = {
     {FT_general, "engine", engine_main, engine_options},
 #endif
     {FT_general, "errstr", errstr_main, errstr_options},
-    {FT_general, "fips-install", fips_install_main, fips_install_options},
+    {FT_general, "fips_install", fips_install_main, fips_install_options},
```

Comment:
I'd rather see `fipsinstall` or `fips-install` than `fips_install`

Reply:
where does this generated name come from?

Reply:
Sigh will change it back to not have the -

Reply:
`apps/progs.pl`, and that's an indication that we shouldn't have dashes in sub-commands.
Basically, it finds all function names ending with `_main` and makes commands from what is before that.

Reply:
> Sigh will change it back to not have the -

But thanks for trying, nevertheless! :-)

Reply:
There is nothing more satisfying than changing a name back and forth..
You should know after playing with the headers :)